apis/python/src/tiledb/vector_search/storage_formats.py

```diff
@@ -1,17 +1,21 @@
+import tiledb
+
 storage_formats = {
     "0.1": {
         "CENTROIDS_ARRAY_NAME": "centroids.tdb",
         "INDEX_ARRAY_NAME": "index.tdb",
         "IDS_ARRAY_NAME": "ids.tdb",
         "PARTS_ARRAY_NAME": "parts.tdb",
         "PARTIAL_WRITE_ARRAY_DIR": "write_temp",
+        "DEFAULT_ATTR_FILTERS": None,
     },
     "0.2": {
         "CENTROIDS_ARRAY_NAME": "partition_centroids",
         "INDEX_ARRAY_NAME": "partition_indexes",
         "IDS_ARRAY_NAME": "shuffled_vector_ids",
         "PARTS_ARRAY_NAME": "shuffled_vectors",
         "PARTIAL_WRITE_ARRAY_DIR": "temp_data",
+        "DEFAULT_ATTR_FILTERS": tiledb.FilterList([tiledb.ZstdFilter(level=1)]),
```

Comment:
Is there a reason we use level=1 and not leave the default (level 3)?

Reply:
No this is just what I experimented with but I don't expect much difference for level 3. 
Changed this to use the default level